notebook/static/notebook/js/notebook.js

```diff
@@ -621,6 +629,7 @@ define(function (require) {
     Notebook.prototype.toggle_cells_marked = function(cells) {
         cells = cells || this.get_cells();
         cells.forEach(function(cell) { cell.marked = !cell.marked; });
+        this.update_marked_status();
```

Comment:
that might be unnecessary now that `.update_marked_status` is triggered  by the `marked_changed.Cell` event.